src/metrics/singlelabelregressionmetrics.jl

```diff
@@ -25,7 +25,7 @@ function singlelabelregressionypred(
 end
 
 function _singlelabelregressionmetrics(
```

Comment:
change function name to regression_metrics:  single label part through multiple dispatch

Reply:
Sounds good.